### ISSUE TITLE
dehydrated dependency sometimes mis-handles account ID

### ISSUE THREAD

**Issue by bahamat** [arekinath/triton-dehydrated#20]
```
 + Requesting new certificate order from CA...
  + ERROR: An error occurred while sending post-request to https://acme-v02.api.letsencrypt.org/acme/new-order (Status 400)

Details:
HTTP/1.1 400 Bad Request
Server: nginx
Content-Type: application/problem+json
Content-Length: 178
Link: <https://acme-v02.api.letsencrypt.org/directory>;rel="index"
Replay-Nonce: 0101uKKVUG1Lz2QX2EghOCxYKOjn69PrtGRaH17-LUdZLDE
Expires: Thu, 22 Aug 2019 20:09:00 GMT
Cache-Control: max-age=0, no-cache, no-store
Pragma: no-cache
Date: Thu, 22 Aug 2019 20:09:00 GMT
Connection: close

{
  "type": "urn:ietf:params:acme:error:malformed",
  "detail": "Malformed account ID in KeyID header URL: \"https://acme-v02.api.letsencrypt.org/acme/acct/\"",
  "status": 400
}
```

This is fixed upstream in lukas2511/dehydrated@be13dcd.

**Comment by bahamat** [comment]
https://cr.joyent.us/6832

I tested this by building an archive, copying it to a failing instance and deploying a new cert.

**Comment by bahamat** [comment]
Fixed in dc5278a.
